### PR TITLE
Enc 2568

### PR DESCRIPTION
**Pull request by brianbegygr**
Add JsonEncompassConfig.
Support loggly.Fatal logging.

### CODE REVIEW

File: GuaranteedRate.Sextant/Properties/AssemblyInfo.cs [hunk 1]
```diff
@@ -32,5 +35,6 @@
 // You can specify all the values or you can default the Build and Revision Numbers 
 // by using the '*' as shown below:
 // [assembly: AssemblyVersion("1.0.*")]
-[assembly: AssemblyVersion("16.2.2.0")]
-[assembly: AssemblyFileVersion("16.2.2.0")]
+
+[assembly: AssemblyVersion("16.2.2.1")]
+[assembly: AssemblyFileVersion("16.2.2.1")]
```
Comment (jeffpsherman) [inline GuaranteedRate.Sextant/Properties/AssemblyInfo.cs]: Shouldn't this be `16.2.3.0`?
The json config is a new feature not a bug fix

Comment (brianbegygr) [inline GuaranteedRate.Sextant/Properties/AssemblyInfo.cs]: Makes sense.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,15 @@
+## v16.2.2.1 / 2016 Sep 9
```
Comment (jeffpsherman) [inline CHANGELOG.md]: Now `16.2.3.0`

**Comment by jeffpsherman** [comment]
👍

**Pull request merged**